config/mocks/iam_users_all.yaml

```diff
@@ -0,0 +1,4 @@
+IAMUsers:
+  include:
+    names_regex:
+      - .*
```

Comment:
I couldn't see anywhere that this mock was used. Is it safe to have it in here as well - e.g. what's the likelihood that we accidentally delete all our IAM users by using this mock?

Reply:
Thanks for :eyes: that :smile: 
Will remove it now.
Remember kids: Copy & Paste is not the best approach for most things :stuck_out_tongue:

Reply:
I removed it on the latest commit :smile: